src/app.service.js

```diff
@@ -226,4 +232,34 @@ export class AppService {
   formatHttpURL(url) {
     return url.replace(/^(?!https?:\/\/)/, "https://");
   }
+
+  getVelogMainContent(bodyElement) {
+    const allElements = Array.from(bodyElement.querySelectorAll("*"));
+
+    const titleElement = allElements.find((element) => {
+      return (
+        typeof element.className === "string" &&
+        element.className.toLowerCase().includes("tbwpx")
```

Comment:
```suggestion
        element.className.toLowerCase().includes(VELOG_TITLE_CLASS_NAME)
```

문자열을 상수로 관리하면 좋을 것 같습니다.
`const VELOG_TITLE_CLASS_NAME = "tbwpx";`

Reply:
해당 사항 반영하였습니다.